scripts.js

```diff
@@ -79,6 +93,11 @@ async function fetchDataDocuments() {
 
         options.year.prepend(option);
     }
+
+    for (let statsObj of tabletop.sheets('stats').all()) {
```

Comment:
I instinctively recoil at variable names that include their type (`Obj`). Perhaps think of a different name?

Reply:
> * Is it actually too resource intensive to compute these stats in JS rather than in a Google Apps script? I would prefer tracking code within the GH repository to the maximum extent possible
It's not the resource intensity that is the problem, but the computation to find the number of states/countries involves a geocode that needs to be run upon every submission to the forms. The Google Apps Script has a onFormSubmit trigger that serves this purpose nicely. I can add the code for it in this repo, but it wouldn't really do much and it'd be a bit of a duplication. I tried to put it in the Google Drive but for some reason Google Apps Scripts don't appear...

> * The arrow on the right doesn't draw the eye. Perhaps we could move the trigger to be around the options panel in the top left, and maybe have a grey background around the button so it's easier to see against complex backgrounds?

I'll give it a try on both the left and the right, but it certainly does need a gray background.